src/OrchardCore/OrchardCore.Users.Core/Services/UserService.cs

```diff
@@ -69,6 +69,13 @@ public async Task<IUser> AuthenticateAsync(string userName, string password, Act
                 return null;
             }
 
+            if (!(user as User).IsEnabled)
+            {
+                reportError(string.Empty, S["The specified account is disabled, please contact an administrator."]);
```

Comment:
Could it be the same error message as line 56, where and account is locker from SignInManager?

`reportError(string.Empty, S["The specified user is not allowed to sign in."]);`

Or change SignIngManager if we have a custom implementation.

Reply:
Otherwise LGTM.